docs/security.html

```diff
@@ -199,6 +198,7 @@
                     <li>ssl.cipher.suites (Optional). A cipher suite is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol.</li>
                     <li>ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1. It should list at least one of the protocols configured on the broker side</li>
                     <li>ssl.truststore.type=JKS</li>
+                    <li>ssl.truststore.password=test1234</li>
```

Comment:
This isn't a Kafka behavior but SSL behavior...perhaps we should link to some documentation instead of recreating the SSL docs?